bzr.go

```diff
@@ -0,0 +1,78 @@
+package vcsstate
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/shurcooL/go/trim"
+)
+
+type bzr struct{}
+
+func (v bzr) DefaultBranch() string {
+	return v.defaultBranch()
+}
+
+func (bzr) Status(dir string) (string, error) {
+	cmd := exec.Command("bzr", "status")
+	cmd.Dir = dir
+
+	out, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	return string(out), nil
+}
+
+func (bzr) Branch(dir string) (string, error) {
+	cmd := exec.Command("bzr", "branch")
+	cmd.Dir = dir
+
+	out, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	return trim.LastNewline(string(out)), nil
+}
+
+// Bazaar uses UUID instead of SHA-1 hashes and is roughtly 60 characters in length
+const bzrRevisionLength = 60
+
+func (v bzr) LocalRevision(dir string) (string, error) {
+	// Alternative: bzr version-info.
```

Comment:
This comment isn't needed.

In the other file, it was used to document an alternative way of doing this, which was useful during initial development. In this case, your alternative is exactly the same as the primary method.